conftest.py

```diff
@@ -1,39 +1,54 @@
-#!/usr/bin/env python
-
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from urlparse import urlparse
+
+from fxapom.fxapom import FxATestAccount
 import pytest
 
 from mocks.marketplace_api import MarketplaceAPI
 from mocks.mock_application import MockApplication
 
 
-@pytest.fixture(scope='function')
-def mozwebqa_devhub_logged_in(request):
-    from pages.desktop.developer_hub.home import Home
-    mozwebqa = request.getfuncargvalue('mozwebqa')
-    dev_home = Home(mozwebqa)
-    dev_home.go_to_developers_homepage()
-    dev_home.login(mozwebqa, user="default")
+@pytest.fixture
+def fxa_test_account(mozwebqa):
+    return FxATestAccount(mozwebqa.base_url).create_account()
+
+
+@pytest.fixture
+def new_user(fxa_test_account):
+    return {'email': fxa_test_account.email,
+            'password': fxa_test_account.password,
+            'name': fxa_test_account.email.split('@')[0]}
+
+
+@pytest.fixture
+def stored_users(variables):
+    return variables['users']
+
+
+@pytest.fixture
+def existing_user(stored_users):
+    return stored_users['default']
+
 
-    return mozwebqa
+@pytest.fixture
+def api(existing_user, mozwebqa):
+    host = urlparse(mozwebqa.base_url).hostname
+    key = existing_user['api'][host]['key']
+    secret = existing_user['api'][host]['secret']
+    return MarketplaceAPI(key, secret, host)
 
 
-@pytest.fixture(scope='function')
-def free_app(request):
-    """Return a free app created via the Marketplace API, and automatically delete the app after the test."""
-    mozwebqa = request.getfuncargvalue('mozwebqa')
-    request.app = MockApplication()
-    api = MarketplaceAPI.get_client(mozwebqa.base_url,
-                                    mozwebqa.credentials)
-    api.submit_app(request.app)
+@pytest.fixture
+def free_app(request, api):
+    """Return a free app created via the Marketplace API."""
```

Comment:
Is there a reason you chose to remove the second part of the docstring? The part about deleting the app after the test?